bin/carthage-bootstrap-if-needed

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+if ! diff Cartfile.resolved Carthage/Cartfile.resolved &>/dev/null; then
+  carthage bootstrap 
+  cp Cartfile.resolved Carthage
+fi
```

Comment:
I don't. I was going to try the default, and tweak it from there if it didn't work.